tools/snp.sh

```diff
@@ -585,7 +620,7 @@ build_base_qemu_cmdline() {
   add_qemu_cmdline_opts "-smp ${GUEST_SMP}"
   add_qemu_cmdline_opts "-m ${GUEST_MEM_SIZE_MB}M"
   add_qemu_cmdline_opts "-no-reboot"
-  add_qemu_cmdline_opts "-vga none"
+  add_qemu_cmdline_opts "-vga std"
```

Comment:
Would it make more sense to add a flag here if we want VGA enabled/disabled? @ryansavino thoughts?